README.md

```diff
@@ -45,48 +46,40 @@ Releases can be found on the [release list][releases].
 
 
 For semi-official builds and third-party packages please see
-[mpv.io](http://mpv.io/installation/).
+[mpv.io/installation](http://mpv.io/installation/).
 
-## Changelog
+## Changelogs
```

Comment:
Naming the section "Changelog" when there are multiple changelogs is misleading. It is the proper plural and shouldn't be avoided just because it is unusual.